spec/openapi.yml

```diff
@@ -1337,7 +1421,12 @@ components:
         endedAt: 2019-05-09T20:05:46.815920Z
         durationMs: 4250894125
         args: {'email': 'me@example.com', 'emailOnFailure': 'false', 'emailOnRetry': 'true', 'retries': '1'}
-
+        facets: {
+          'context': {
+            '_producer': 'https://github.com/OpenLineage/OpenLineage/blob/v1-0-0/client',
+            '_schemaURL': 'https://github.com/OpenLineage/OpenLineage/blob/v1-0-0/spec/OpenLineage.yml#MyCustomJobFacet'
```

Comment:
Looking over the OpenLineage spec, should these facets b part of a run?

Reply:
do you mean, adding a level of nesting?
To me, it makes sense to add the run facets as part of the run event here.

Reply:
I see `_producer ` and `_schemaURL` are base facets within the spec.

Reply:
It wasn't clear on how `_producer` and `_schemaURL` were used within the facet (which I know have clarity on). Yep, adding `facets` to a run definitely makes sense here 👍